src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1078 @@ metadata_pkg.(
+"""
+BayesianLDA
+"""
+$(MMI.doc_header(SubspaceLDA))
+
+`SubspaceLDA`: Multiclass linear discriminant analysis. Suitable for high
+dimensional data (Avoids computing scatter matrices `Sw` ,`Sb`). The algorithm learns a
+projection matrix `P = W*L` that projects a feature matrix `Xtrain` onto a lower
+dimensional space of dimension `nc - 1` such that the trace of the transformed
+between-class scatter matrix(`Pᵀ*Sb*P`) is maximized relative to the trace of the
+transformed within-class scatter matrix (`Pᵀ*Sw*P`). The projection matrix is scaled
+such that `Pᵀ*Sw*P = mult*I` or `Pᵀ*Σw*P=mult/(n-nc)*I` (where `n` is the number of
+training samples, mult` is  one of `n` or `1` depending on whether `Sb` is normalized,
+`Σw` is the within-class covariance matrix, and `nc` is the number of unique classes
+in `y`) and also obeys `Wᵀ*Sb*p = λ*Wᵀ*Sw*p`, for every column `p` in `P`.
+Predicted class posterior probability for feature matrix `Xtest` are derived by
+applying a softmax transformation to a matrix `Pr`, such that  rowᵢ of `Pr` contains
+computed distances(based on a distance metric) in the transformed space of rowᵢ in
+`Xtest` to the centroid of each class.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitypes with `schema(X)`
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `<:OrderedFactor(2)` or `<:Multiclass(2)`; check the scitype
+  with `scitype(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Hyper-parameters
+
+- `normalize=true`: Option to normalize the between class variance for the number of
+   observations in each class, one of `true` or `false`.
+- `out_dim`: The dimension of the transformed space to be used by `predict` and
+   `transform` methods, automatically set if 0 is given (default).
+- `dist=SqEuclidean`: The distance metric to use when performing classification
+   (to compare the distance between a new point and centroids in the transformed space),
+   an alternative choice can be the `CosineDist`.
+
+
+# Operations
+
+- `transform(mach, Xnew)`: Return a lower dimentional projection of the input `Xnew` having the same scitype as `X` above.
+- `predict(mach, Xnew)`: Return predictions of the target given
+  features `Xnew` having the same scitype as `X` above. Predictions
+  are probabilistic.
+- `predict_mode(mach, Xnew)`: Return the modes of the probabilistic predictions
+   returned above.
+
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+BUG: RENAME, Make note on top of PR for inconsistency
+- `class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `projection_matrix`: The matrix used to project `X` into a lower dimensional space.
+
```

Comment:
Please revise these two fields as per vanilla LDA.

Not sure what you mean by BUG here.

Reply:
FYI Not yet done